src/index.js

```diff
@@ -13,9 +13,12 @@ var closingDelayMs = 1000;
 session.clean();
 
 var telemetry = {
-  version: version,
   name: 'auth0-cordova',
-  lib_version: auth0.version
+  version: version,
+  env: {
+    'auth0.js': '9.4.2',
```

Comment:
I'm guessing this should use `auth0.version` and not be hard-coded.

Reply:
@joshcanhelp Not sure how I missed this - I've fixed it now